cwltool/main.py

```diff
@@ -713,6 +713,7 @@ def my_represent_none(self, data):  # pylint: disable=unused-argument
                                      logger=_logger)
 
             if out is not None:
+                runtimeContext.research_obj.create_job(None, out, True)
```

Comment:
`runtimeContext.research_obj` could be `None` here

Reply:
Thanks :) fixed it in new commit